.github/workflows/ci.yml

```diff
@@ -8,6 +8,8 @@ on:
     branches: [ main, master, beta, Release2.7, release-3.0 ]
   pull_request:
     branches: [ main, master, beta, Release2.7, release-3.0 ]
+  merge_group:
```

Comment:
these changes seem odd as they are from my PR's...

Can you remove all but your changes